src/dom.js

```diff
@@ -926,6 +926,21 @@ export function toggleAttribute(element, name, forced) {
   return enabled;
 }
 
+/**
+ * Parses a string as a boolean value using the expanded rules for DOM boolean
+ * attributes:
+ * - a `null` or `undefined` returns `null`;
+ * - an empty string returns `true`;
+ * - a "false" string returns `false`;
+ * - otherwise, `true` is returned.
```

Comment:
It's worth noting that according to [the standard](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes), `attr="asd"` would return `false` whereas here it would return `true`. Considering we are breaking away from it already by accepting `"true"` differently, I'm not sure how critical this is to follow, but thought I'd mention it here.

Reply:
I can't seem to derive `attr=asd -> false` from the spec. Could you pls confirm?

Reply:
Discussed offline. It looks like spec is saying that any non-empty and not-equal-to-attribute-name values are invalid, but no explanation on what value that'd yield. The empirical result is `true`.